src/lib/core/types/visualization.ts

```diff
@@ -61,8 +61,7 @@ export const ComputationConfiguration = intersection([
     collectionVariable: VariableDescriptor,
   }),
   partial({
-    alphaDivMethod: string,
-    rankingMethod: string,
+    method: string,
```

Comment:
I think this is causing 422s in the visualizations. For right now the two apps with computations need a type of method (though called different things) and a collection variable. In the future, we'll have an app with different data or even no data as input, and then 2 or more methods. So though i'd love to make it as general as a simple "method" here like you've done, unfortunately we won't be able to sustain the simplicity even for the next two apps